packages/utils/src/lib/report-to-md.ts

```diff
@@ -105,73 +137,46 @@ function reportToCategoriesSection(report: ScoredReport): string {
       categoryDocs +
       categoryScore +
       NEW_LINE +
+      groups +
       NEW_LINE +
-      refs
+      audits
     );
   }, '');
 
   return h2('🏷 Categories') + NEW_LINE + categoryDetails;
 }
 
-function auditRefItemToCategorySection(
-  refSlug: string,
-  refPlugin: string,
+function auditItemToCategorySection(
+  audit: WeighedAuditReport,
   plugins: ScoredReport['plugins'],
 ): string {
-  const plugin = plugins.find(({ slug }) => slug === refPlugin) as PluginReport;
-  const pluginAudit = plugin?.audits.find(({ slug }) => slug === refSlug);
-
-  if (!pluginAudit) {
-    throwIsNotPresentError(`Audit ${refSlug}`, plugin?.slug);
-  }
-
   const auditTitle = link(
-    `#${slugify(pluginAudit.title)}-${slugify(plugin.title)}`,
-    pluginAudit?.title,
+    `#${slugify(audit.title)}-${slugify(audit.plugin)}`,
```

Comment:
This should slugify the plugin title:

```suggestion
    `#${slugify(audit.title)}-${slugify(getPluginNameFromSlug(
      audit.plugin,
      plugins,
    ))}`,
```

(:top: Probably best to assign result of `getPluginNameFromSlug` to a variable, since you reuse it.)

Reply:
Reused the plugin title